gdx/src/com/badlogic/gdx/graphics/g3d/particles/batches/BillboardParticleBatch.java

```diff
@@ -398,15 +406,20 @@ private void fillVerticesGPU (int[] particlesOffset) {
 				float a = colorChannel.data[colorOffset +ParticleChannels.AlphaOffset];
 				float cosRotation = rotationChannel.data[rotationOffset +ParticleChannels.CosineOffset];
 				float sinRotation = rotationChannel.data[rotationOffset +ParticleChannels.SineOffset];
-
+				// Transform the particle position (and optionally scale) by the world matrix of its controller.
+				pos.mul(worldTransform);
+				if (data.controller.worldTransformScalesParticles) {
+					sx *= worldViewTransform.getScaleX();
```

Comment:
Best to cache the result of getScaleX() etc, for the scale on each axis rather than call the method for each particle. Likewise for the rest of this PR where this occurs.